test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ControllerActionArgumentBinderTests.cs

```diff
@@ -361,7 +361,7 @@ public class ControllerActionArgumentBinderTests
             binder
                 .Setup(b => b.BindModelAsync(It.IsAny<ModelBindingContext>()))
                 .Returns(Task.FromResult(
-                    result: new ModelBindingResult(model: null, key: string.Empty, isModelSet: true)));
+                    result: new ModelBindingResult(model: null, key: string.Empty, isModelSet: false)));
```

Comment:
Not really much at all. Note I didn't change the assertions at all. My thought was to avoid testing a corner case in this general test. Few if any model binders return a result with `Model == null && IsModelSet`. But I can revert this one-liner change or rename the test if you prefer.